app/http.php

```diff
@@ -0,0 +1,313 @@
+<?php
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+use Swoole\Atomic;
+use Swoole\Coroutine\Http\Client;
+use Utopia\Logger\Log;
+use Utopia\Logger\Logger;
+use Swoole\Coroutine\Http\Server;
+use Swoole\Http\Request as SwooleRequest;
+use Swoole\Http\Response as SwooleResponse;
+use Swoole\Runtime;
+use Swoole\Table;
+use Swoole\Timer;
+use Utopia\App;
+use Utopia\Balancing\Algorithm\Random;
+use Utopia\Balancing\Algorithm\RoundRobin;
+use Utopia\Balancing\Balancing;
+use Utopia\Balancing\Option;
+use Utopia\CLI\Console;
+use Utopia\Logger\Adapter;
+
+Runtime::enableCoroutine(true, SWOOLE_HOOK_ALL);
+
+$executorStates = new Table(1024);
```

Comment:
Does this limit us to 1024 executors?

Reply:
Correct. I will calculate amount of executors in ENV variable instead.